cmd/cluster-agent/app/app.go

```diff
@@ -416,3 +433,23 @@ func setupClusterCheck(ctx context.Context) (*clusterchecks.Handler, error) {
 	log.Info("Started cluster check Autodiscovery")
 	return handler, nil
 }
+
+func initializeRemoteConfig(ctx context.Context) (*remote.Client, error) {
+	configService, err := remoteconfig.NewService()
+	if err != nil {
+		return nil, fmt.Errorf("unable to create remote-config service: %v", err)
+	}
+
+	if err := configService.Start(ctx); err != nil {
+		return nil, fmt.Errorf("unable to start remote-config service: %v", err)
+	}
+
+	rcClient, err := remote.NewClient("cluster-agent", configService, version.AgentVersion, []data.Product{}, time.Second*5)
```

Comment:
This value is not configurable externally in the core agent at this time. We could use a constant but this is the only place this value is used and I tend to lean on preferring the explicitness in that situation rather than requiring a hop in the file to figure out what the value actually is.